cmd/cluster-agent-cloudfoundry/main.go

```diff
@@ -0,0 +1,39 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build !windows
+
+//go:generate go run ../../pkg/config/render_config.go dcacf ../../pkg/config/config_template.yaml ../../cloudfoundry.yaml
+
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	_ "expvar"         // Blank import used because this isn't directly used in this file
+	_ "net/http/pprof" // Blank import used because this isn't directly used in this file
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/telemetry"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	"github.com/DataDog/datadog-agent/cmd/cluster-agent-cloudfoundry/app"
+)
+
+func main() {
+	// Expose the registered metrics via HTTP.
+	http.Handle("/metrics", telemetry.Handler())
```

Comment:
Just a question here: What is this endpoint for?

Reply:
C&P error from DCA for kubernetes, this won't be need on CF, will remove it now. Thanks for noticing this!